### ISSUE TITLE
USB Breakout?

### ISSUE THREAD

**Issue by Lugoues** [Bengt/AL-FanControl#1]
Should probably add a usb breakout for this, no?

**Comment by Lugoues** [comment]
Ah,  I see. I should just use a Pro Micro and no usb breakout needed! :)

**Comment by Bengt** [comment]
Yes, the Pro Micro acts as an interface between USB and PWM. I uploaded [a photo of the board I am using](https://github.com/Bengt/AL-FanControl/blob/master/fritzing/IMG_20151219_143207_cut.jpg) since about a year, now. Be advised though, that it outputs a 12 V PWM signal, although the PWM specification says it should be 5 V. This was an misunderstanding I made at the time. As it turns out, my fans are tolerant to 12 V PWM inputs, so I never bothered to revise the board and Fritzing. The circuit can be made much simpler, if one uses the 5 V outputs of the Pro Micro directly, so patches are definitely welcome!
